pro/src/routes/OfferEducationalStockEdition/adapters/getEducationalStockAdapter.ts

```diff
@@ -0,0 +1,35 @@
+import * as pcapi from 'repository/pcapi/pcapi'
+
+import { StockResponse } from '../types'
+
+type IPayloadSuccess = {
+  stock: StockResponse | null
+}
+type IPayloadFailure = { stock: null }
+type GetEducationalStockAdapter = Adapter<
+  string,
+  IPayloadSuccess,
+  IPayloadFailure
+>
+
+const FAILING_RESPONSE: AdapterFailure<IPayloadFailure> = {
+  isOk: false,
+  message: 'Nous avons rencontré un problème lors du chargemement des données',
+  payload: { stock: null },
+}
+
+export const getEducationalStockAdapter: GetEducationalStockAdapter =
+  async offerId => {
+    try {
+      const { stocks } = (await pcapi.loadStocks(offerId)) as {
+        stocks: StockResponse[]
+      }
+      return {
+        isOk: true,
+        message: '',
+        payload: { stock: stocks.length > 0 ? stocks[0] : null },
```

Comment:
peut-être que ça ça marche : 
```suggestion
        payload: { stock: stocks[0] ?? null },
```